homework.md

```diff
@@ -92,13 +93,13 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save your code onto a repository
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio Code
 
 ## Question 17
```

Comment:
Hi Sinombulelo.
 Those are so pretty  Awesome detailed answers, however I've noticed that you haven't completed Question #17 and #18 You have not Inputted your Own Answers . You might want to take a look at it?

Reply:
Hi Lawrence!

I figured since the answers are correct, I'd leave them be but I will make them my own. Thanks!